lightspeed/assemblies/assembly_generating-playbooks.adoc

```diff
@@ -0,0 +1,28 @@
+ifdef::context[:parent-context: {context}]
+
+:_content-type: ASSEMBLY
+
+[id="generating-playbooks_{context}"]
+
+= Generating playbooks and viewing playbook explanations
+
+:context: generating-playbooks
+
+[role="_abstract"]
+
+Using the Ansible VS Code extension, you can create Ansible playbooks using a natural language interface in English. {LightspeedshortName} reads the natural language prompts, identifies the context for the playbook, and generates an entire playbook recommendation based on your task intent. You can also view the explanations for new or existing playbooks. The playbook explanations describe what the playbook will do and contextualize its impact.
```

Comment:
what the playbook does and contextualizes its impact.

Reply:
This is done, thank you